packages/data-context/src/actions/ElectronActions.ts

```diff
@@ -13,6 +14,7 @@ export interface ElectronApiShape {
   copyTextToClipboard(text: string): void
   isMainWindowFocused(): boolean
   focusMainWindow(): void
+  createNotification(title: string, body?: string, icon?: string): Notification
```

Comment:
Slight disagreement on typings - looks like we treat `body` as optional here, but calling functions default to empty string. Any reason not to align one way or the other?